packages/nextjs/app/builders/0xF11046328814004583c8e74050f0d59520f53e90/page.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+interface ProfileLink {
+  name: string;
+  url: string;
+}
+
+const profileLinks: ProfileLink[] = [
+  { name: "GitHub", url: "https://github.com/Welith" },
+  { name: "Twitter", url: "https://twitter.com/bo4ka7a" }, // Replace with URL
+];
+
+const Home: React.FC = () => {
+  return (
+    <div style={{ textAlign: "center", paddingTop: "50px" }}>
```

Comment:
We are using [Tailwind](https://tailwindcss.com/), so you don't need to use inline styles.

Reply:
Same for the other inlined styles.